Documentation/getting-started.md

```diff
@@ -2,6 +2,7 @@
 
 This guide will walk you through a deployment of a single-master/multi-worker Kubernetes cluster on CoreOS. We're going to:
 
+- You must use a CoreOS release 773.1.0+ for the `kubelet` to be present in the image
```

Comment:
Nit: This seems out of place. The heading is "We're going to", followed by "You must use...".

This also isn't true of etcd dedicated etcd nodes (can be run from stable). We note the required version in both the master installation and the worker installation steps. Can we just drop this line for now to get this merged (unless you have another placement suggestion)?